src/main/java/io/confluent/castle/role/Schema.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.confluent.castle.role;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class Schema {
+    private final String subject;
+    private final String schema;
+
+    @JsonCreator
+    public Schema(@JsonProperty("subject") String subject,
+                  @JsonProperty("schema") String schema) {
```

Comment:
This could just be a `JsonObject` rather than a `String`, right?  Then you wouldn't need to do any escaping or whatever, but just copy the JSON object directly to the config file(s) using Jackson.

Reply:
With Avro, it is JSON, but with Protobuf (I'm working on adding Protobuf support to Schema Registry) it will be Protobuf IDL, which is not JSON.

Reply:
ok